src/Process/GlobeTileProcessing.js

```diff
@@ -133,9 +96,14 @@ export function globeSubdivisionControl(minLevel, maxLevel, sseThreshold, maxDel
             return false;
         }
 
-        const sse = computeNodeSSE(context.camera, node);
-
-        return SSE_SUBDIVISION_THRESHOLD < sse;
+        node.sse = ScreenSpaceError.computeFromBox3(
```

Comment:
why you replace the bounding sphere with OBBs?

Reply:
Because the OBB is more closer to the real volume of the `node` then a sphere.